sce.py

```diff
@@ -6,24 +6,28 @@
 from tools.sce_presubmit_handler import ScePresubmitHandler
 
 parser = argparse.ArgumentParser()
-parser.add_argument(
-    'command', help='Setup for the SCE tool to run ' +
-    '(setup, run, presubmit, etc.)')
-parser.add_argument(
-    '--project', '-p', nargs='+', help='Project to run presubmit checks for.')
-parser.add_argument(
-    '--proto', nargs=1,
-    help='The language(s) to generate proto code.')
-parser.add_argument(
-    '--language', nargs='+',
-    help='The language(s) to generate proto code.')
-parser.add_argument(
+sub_parser = parser.add_subparsers(required=True, dest='command')
+
+setup_parser = sub_parser.add_parser('setup', help='Setup the sce tool')
+
+run_parser = sub_parser.add_parser('run', help='Run an sce service')
+run_parser.add_argument(
     '--service', '-s', nargs='*',
-    help='SCE Service name')
-parser.add_argument(
+    help='SCE Service name'
+)
+run_parser.add_argument(
     '--dbpath',
     help='Specify a path for the volume used by MongoDB.'
 )
+
+presubmit_parser = sub_parser.add_parser('presubmit', help='Project to run presubmit checks for')
+presubmit_parser.add_argument(
+    '--project', '-p', nargs='+',
+    help='Project to run presubmit checks for.'
+)
+
+test_parser = sub_parser.add_parser('test')
```

Comment:
i know it's unrelated but can we remove the test command for now? all it does is attempt to add the alias but we dont document it anywhere.

basically remove this line and the ` elif args.command == 'test':` block at the bottom of this file

Reply:
Done with a3dd9ce5b975187012ef0d9ff68f3ccc89f60a4c.